content/stratify.md

```diff
@@ -0,0 +1,169 @@
+needed:
+
+  * Partition disks as required for EFI or BIOS boot
+  * Install packages needed for Stratis
+  * Create a Stratis pool and filesystem
+  * Install packages into the new root filesystem
+  * Install Grub2 or another bootloader on the installation disk
+  * Generate an initramfs image with the required components
+  * Configure the bootloader to boot the new system
+
+This can be done manually but it is a long and tedious process with plenty of
+opportunity for mistakes to creep in. A single error in any of the above steps may
+lead to an unbootable system and user frustration.
+
+
+### Introducing stratify.py
+
+In this post we'll look at using the `stratify.py` script to automate the
+installation of a virtual machine using Stratis as the root filesystem. The script
+automates the preparation of the basic system layout and then runs Anaconda to
+perform the OS installation. Once the installer has done its work `stratify.py` will
+set up the boot loader and other details needed for a working system.
+
+Since Anaconda is used to carry out the OS installation we will use a [kickstart
+file](https://pykickstart.readthedocs.io/en/latest/) to configure the installed
+system.
+
+Refer to the [stratify documentation](https://github.com/bmr-cymru/stratify) for more
+details or press on to get started right away!
+
+
+### Requirements
+
+In order to install a system with Stratis as the root filesystem you will need an
+`x86_64` virtual machine configured with the following specification:
+
+  * At least 3GiB memory
+  * BIOS or UEFI firmware
+  * One 10GiB or larger VirtIO disk for the installation
+  * The [Fedora LiveCD media](https://fedoraproject.org/workstation/download/) (Fedora 37 or 38)
+
+
+### Getting started
+Configure the machine with your preferred virtualization solution and then boot the
+Live desktop environment. Once the system has booted dismiss the "Install to hard
+disk" dialog, open a terminal window and switch to the root account by running `su -`.
+
+Download the `stratify.py` script and example kickstart file from GitHub:
+
+```
+# wget https://raw.githubusercontent.com/bmr-cymru/stratify/main/stratify.py
+# wget https://raw.githubusercontent.com/bmr-cymru/stratify/main/ks.cfg
+```
+
+The kickstart file can be customized before beginning the installation - for example
+to set the system language, timzeone, root account password or package selection for
+the install. See the [kickstart
+documentation](https://pykickstart.readthedocs.io/en/latest/) for further
+information.
+
+To set the root account password run the following command and edit the kickstart
+file to set the new value:
+
+```
+$ openssl passwd -6 "newpassword"
+$6$ly/YwB6/b5R6Pezz$zZGzBKgkPZvYHtefBwC7gJKB0BpW7ANJyt0sfjIf46yKS2IGfcnhfFW6wiXJLpYeQXvIBlJG/W7wukX0/S9Wu/
+```
+
+In `ks.cfg`:
+
+```
+# Root password
+rootpw --iscrypted $6$ly/YwB6/b5R6Pezz$zZGzBKgkPZvYHtefBwC7gJKB0BpW7ANJyt0sfjIf46yKS2IGfcnhfFW6wiXJLpYeQXvIBlJG/W7wukX0/S9Wu/
+```
+
```

Comment:
In my testing I found that the ks.cfg needs to be copied to /root prior to running stratify.py on Fedora 38 otherwise the stratify.py will fail with "Kickstart file /root/ks.cfg is missing".    There could be a step here to 
let the user know to run   cp ks.cfg /root/

Reply:
During my test run, after the Live CD desktop environment started, I opened a terminal and executed `sudo su -`; this results in a root shell that is in root's home directory (/root).